protopipe_grid_interface/scripts/submit_jobs.py

```diff
@@ -181,6 +189,8 @@ def main():
     config_file = cfg["General"]["config_file"]
     mode = cfg["General"]["mode"]  # One mode naw
     particle = cfg["General"]["particle"]
+    if "particle" in switches:
```

Comment:
if we check that it is correctly initialized this conditional becomes superfluous because we exit with an error at the  initialization of the script's arguments

Reply:
we should at least make sure that particle is always specified on top of what you want to do here because I don't think a default makes much sense for this option